Library/Homebrew/cli_parser.rb

```diff
@@ -4,10 +4,17 @@
 module Homebrew
   module CLI
     class Parser
+      # rubocop:disable Style/ClassVars
+      @@args = OpenStruct.new
+      # rubocop:enable Style/ClassVars
       def self.parse(&block)
         new(&block).parse
       end
 
+      def self.args
+        @@args
```

Comment:
You can use `@args ||= OpenStruct.new` here instead of the class variable.

Reply:
This is another option but I do think a global `Homebrew.args` is nice and snappy.